pkg/cloudprovider/providers/azure/azure_managedDiskController.go

```diff
@@ -173,3 +190,13 @@ func (c *ManagedDiskController) ResizeDisk(diskName string, oldSize resource.Qua
 
 	return newSizeQuant, nil
 }
+
+// get resource group name from a managed disk URI, e.g. return {group-name} according to
+// /subscriptions/{sub-id}/resourcegroups/{group-name}/providers/microsoft.compute/disks/{disk-id}
```

Comment:
be more specific, add link https://docs.microsoft.com/en-us/rest/api/compute/disks/get

Reply:
fixed